### ISSUE TITLE
So what's the design philosophy behind your version?

### ISSUE THREAD

**Issue by JordanPaoletti** [Dmoro/ProjectLife#1]
It seems any placement creates a green explosion which then fills everything with red.

**Comment by danielemoro** [comment]
Yes, but try putting it on max speed and waiting a few minutes. Sometimes really interesting patterns emerge. I'm still working on making it more interesting, and I will eventually add more user control and some tutorial to make sense of all the colors. I just need a time machine to do everything :D
